src/scheduled_tasks.rs

```diff
@@ -35,19 +44,20 @@ use std::time::Duration;
 use tracing::{error, info, warn};
 
 /// Schedules various cleanup tasks for lemmy in a background thread
-pub async fn setup(context: LemmyContext) -> LemmyResult<()> {
+pub async fn setup(context: Data<LemmyContext>) -> LemmyResult<()> {
   // Setup the connections
   let mut scheduler = AsyncScheduler::new();
   startup_jobs(&mut context.pool()).await;
 
-  let context_1 = context.clone();
-  // Update active counts every hour
+  let context_1 = context.reset_request_count();
+  // Update active counts expired bans and unpublished posts every hour
   scheduler.every(CTimeUnits::hour(1)).run(move || {
-    let context = context_1.clone();
+    let context = context_1.reset_request_count();
 
     async move {
       active_counts(&mut context.pool()).await;
       update_banned_when_expired(&mut context.pool()).await;
+      publish_scheduled_posts(&context).await;
```

Comment:
This one's tough. Maybe every 15m would work for now?

Reply:
Done